MEASURE.md

```diff
@@ -0,0 +1,236 @@
+  "scoring": {
+    "coding": [{
+      "system": "http://terminology.hl7.org/CodeSystem/measure-scoring",
+      "code": "proportion"
+    }]
+  },
+  "group": [ {
+    "id": "males",
+    "population": [ {
+      "id": "initial-population",
+      "code": {
+        "coding": [ { 
+          "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+          "code": "initial-population"
+      } ]
+      },
+      "criteria": {
+        "language": "text/cql-identifier",
+        "expression": "Patients"
+      }
+    }, {
+      "id": "denominator",
+      "code": {
+        "coding": [ { 
+          "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+          "code": "denominator"
+        } ]
+      },
+      "criteria": {
+        "language": "text/cql-identifier",
+        "expression": "Group Member"
+      }
+    }, {
+      "id": "numerator",
+      "code": {
+        "coding": [ { 
+          "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+          "code": "numerator"
+        } ]
+      },
+      "criteria": {
+        "language": "text/cql-identifier",
+        "expression": "Males"
+      }
+    } ],
+      "stratifier": [ {
+        "id": "by-age",
+        "criteria": {
+            "language": "text/cql-identifier",
+            "expression": "Age Stratifier"
+        }
+      }, 
+      { ... ... ... } ]
+  }, 
+  { ... ... ... } ],
+  "supplementalData": [
+      {
+          "description": "groups",
+          "criteria": {
+              "language": "text/cql-identifier",
+              "expression": "Group"
+          }
+      }
+  ]
+}
+```
+
+Some notable fields
+
+**url**: The complete url of Measure with measure.name i.e. http://fhir.org/guides/who/anc-cds/Measure/HOUSEHOLDIND01
+
+**name**: A unique name for Measure i.e. HOUSEHOLDIND01. The Measure is loaded by name and url into measure processor
+
+**relatedArtifact**: Helper libraries to load before running Measure to be used by measure logic libraries i.e. FHIRHelpers|4.0.1. The Library is loaded by its canonical url i.e. Library/123
+
+**library**: The CQL logic libraries used by Measure for calculation i.e. http://fhir.org/guides/cqf/common/Library/HOUSEHOLDIND01. The logic library is loaded by its url. The url must end with /Library/name 
+
+**scoring**: proportion | ratio | continuous-variable | cohort
+
+**group**: The section of report defining the stats for a specific indicator (may be disaggregated by stratifier). Each group has following components
+- id: Group name/id.
+- population: The calculations (or name of variable in CQL that defines the value) for each of population components i.e. initial-population, denominator, numerator
+- stratifier: The disaggregations for given population numerator. i.e. by age, by month, by education etc 
+- supplementalData: Any extra data or intermediate calculation to be output to final report
+
+
+## MEASURE LIBRARY
```

Comment:
I think this Title should read CQL Expression containing decision logic or Measure to compute